src/Table/AllDataTable.js

```diff
@@ -290,7 +290,8 @@ const AllDataTable = (props) => {
           data.address === "-999" ||
           data.latlng === "-999" ||
           data.notes === "-999" ||
-          data.state === "-999"
+          data.state === "-999" ||
+          data.protocols_enrolled === "-999"
```

Comment:
Same story as above. Filtering should only depend on this `protocols_enrolled` column. The others should be displayed (and may be a clue to the user that they should unenroll that site!)